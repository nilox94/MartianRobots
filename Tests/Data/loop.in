3 3
0 0 E
FFFLFFFLFFFLFFFL
0 0 E
FFFLFFFLFFFLFFFLFFFLFFFLFFFLFFFL
0 0 N
FFFRFFFRFFFRFFFR
0 0 N
FFFRFFFRFFFRFFFRFFFRFFFRFFFRFFFR
