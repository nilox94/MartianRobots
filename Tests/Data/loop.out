0 0 E
0 0 E
0 0 N
0 0 N
